src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAICitizenWander.java

```diff
@@ -91,14 +92,14 @@ private double getValidHeight(@NotNull final Vec3d position)
             returnHeight = 0;
         }
 
-        while (returnHeight >= 1 && citizen.worldObj.isAirBlock(new BlockPos(MathHelper.floor_double(position.xCoord),
+        while (returnHeight >= 1 && CompatibilityUtils.getWorld(citizen).isAirBlock(new BlockPos(MathHelper.floor_double(position.xCoord),
                                                                               (int) returnHeight,
                                                                               MathHelper.floor_double(position.zCoord))))
         {
             returnHeight -= 1.0D;
         }
 
-        while (!citizen.worldObj.isAirBlock(new BlockPos(MathHelper.floor_double(position.xCoord), (int) returnHeight, MathHelper.floor_double(position.zCoord))))
+        while (!CompatibilityUtils.getWorld(citizen).isAirBlock(new BlockPos(MathHelper.floor_double(position.xCoord), (int) returnHeight, MathHelper.floor_double(position.zCoord))))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 182 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)